### ISSUE TITLE
INFO:tensorflow:loss = 0.0, mrr = nan, step = 0

### ISSUE THREAD

**Issue by PCML-WangZhe** [alibaba/euler#147]
有人遇到过这种问题吗

**Comment by sadhen** [comment]
我遇到了一样的问题，这是因为你的数据格式不正确。

你看一下代码，就可以看到：

``` python
  num_steps = int((flags_obj.max_id + 1) // flags_obj.batch_size *
                   flags_obj.num_epochs)
  hooks.append(tf.train.StopAtStepHook(last_step=num_steps))
```

**Comment by PCML-WangZhe** [comment]
> 我遇到了一样的问题，这是因为你的数据格式不正确。
> 
> 你看一下代码，就可以看到：
> 
> ```python
>   num_steps = int((flags_obj.max_id + 1) // flags_obj.batch_size *
>                    flags_obj.num_epochs)
>   hooks.append(tf.train.StopAtStepHook(last_step=num_steps))
> ```
> 
> 你的max_id没有设置的话，就是-1，所有num_steps=0。

感谢回复，你说的是参数格式的问题吧..可是我确实设置了max_ids，可能我描述的不清楚，我的num_step在日志是变的，不是一直是0，但是loss一直是0，mrr一直是nan

**Comment by sadhen** [comment]
OK 这样的话，和我遇到的问题是不一样的
